src/ticketmap/index.js

```diff
@@ -372,88 +370,88 @@ export default class TicketMap extends Component<*, State> {
       if (this.venueSections.includes(section)) {
         return this.doHoverCleanup(section)
       }
     } else if (target !== this.rootRef) {
-      return this.onMouseOut({ target: target.parentNode });
+      return this.onMouseOut({ target: target.parentNode })
     }
   }
 
   onClick = ({ target }: any) => {
     if (target.hasAttribute('data-section-id')) {
-      const section = target.getAttribute('data-section-id');
+      const section = target.getAttribute('data-section-id')
       if (this.venueSections.includes(section)) {
-        return this.selectSectionOrZone(section);
+        return this.selectSectionOrZone(section)
       }
     } else if (target !== this.rootRef) {
-      return this.onClick({ target: target.parentNode });
+      return this.onClick({ target: target.parentNode })
     }
   }
 
   /**
    * Interactions
    */
 
-  doHover(tooltipX: any, tooltipY: any, section: string): void {
-    const { zone } = this.state.sectionZoneMapping[section];
+  doHover (tooltipX: any, tooltipY: any, section: string): void {
+    const { zone } = this.state.sectionZoneMapping[section]
 
     const newState = {
       activeTooltip: true,
       tooltipX,
       tooltipY,
       tooltipSectionName: section
-    };
+    }
 
     if (this.state.isZoneToggled) {
-      newState.currentHoveredZone = zone;
-      this.highlightZone(zone);
+      newState.currentHoveredZone = zone
+      this.highlightZone(zone)
     } else {
-      newState.currentHoveredSection = section;
-      this.highlightSection(section);
+      newState.currentHoveredSection = section
+      this.highlightSection(section)
     }
 
-    this.setState(newState);
+    this.setState(newState)
   }
 
-  doHoverCleanup(section: string): void {
+  doHoverCleanup (section: string): void {
     this.setState({ activeTooltip: false })
 
     if (this.state.isZoneToggled) {
-      return this.unhighlightZone(this.state.sectionZoneMapping[section].zone);
+      return this.unhighlightZone(this.state.sectionZoneMapping[section].zone)
     }
 
-    this.unhighlightSection(section);
+    this.unhighlightSection(section)
   }
 
-  selectSectionOrZone(section: string): void {
+  selectSectionOrZone (section: string): void {
     if (this.state.isZoneToggled) {
-      const { zone } = this.state.sectionZoneMapping[section];
-      this.toggleZoneSelect(zone, !this.areAllSectionsInTheZoneSelected(zone));
+      const { zone } = this.state.sectionZoneMapping[section]
+      this.toggleZoneSelect(zone, !this.areAllSectionsInTheZoneSelected(zone))
     } else {
-      this.toggleSectionSelect(section, !this.state.selectedSections.includes(section));
+      this.toggleSectionSelect(section, !this.state.selectedSections.includes(section))
     }
   }
 
-  render(): ?React$Element<any> {
+  render (): ?React$Element<any> {
     return (
       <div
-        ref={element => this.rootRef = element}
-        onMouseOver={this.onMouseOver.bind(this)}
```

Comment:
Why are we removing bindings?

Reply:
We aren't?

Reply:
It's already bound via the `onMouseOver = () => {}`. We would need to bind it if it was `onMouseOver() {}`.